examples/backends/sglang/launch/disagg_grpc.sh

```
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Disaggregated prefill/decode via the Rust bridge to SGLang's gRPC server.
+# Five processes: dynamo.frontend, sglang prefill, sglang decode, bridge prefill, bridge decode.
+# GPUs: 2
+
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)